api/views.py

```diff
@@ -257,5 +265,19 @@ def checkAuthentication(request):
     return Response({"is_superuser": request.user.is_superuser}, status=status.HTTP_200_OK)
 
 
+@api_view([GET])
+def general_honeypot_view(request):
+    active = request.query_params.get("onlyActive")
```

Comment:
Add a log here: in this way in case this endpoint fail and the last log isn't logged we know a request has been made.